data/portals.csv

```diff
@@ -918,3 +918,4 @@ Transport for NSW is a state government agency in Australia.","transportation,lo
 agridata-tn,Agriculture Portal in Tunisia,http://www.agridata.tn/,Tunisia Ministry of Agriculture,Tunisia Ministry of Agriculture,,Government,Agriculture data portal from the Tunisian Ministry of Agriculture,"agriculture,tn",,,,Tunisia,"36.8178, 10.1795",TN,fr,active,2021-11-04T03:32:54,,,,
 open_data_ps,Palestine Open Data Portal,http://www.opendata.ps/,Ministry of Telecom and IT,Ministry of Telecom and IT,2021-10-12,Government,A platform that offers (mostly) government and other institutions data in accordance with the Open Data Policy in Palestine - a framework for publishing (non-private) data that is open (technically and legally).,"palestine,ps",cc-by,https://opendefinition.org/licenses/cc-by/,,Palestine,"31.916873936677476, 35.213386165348155",PS,ajp,active,2022-02-11T02:56Z,CKAN,http://www.opendata.ps/api/3,CKAN API,
 portal-dados-urbanos-uff,Portal de Dados Urbanos PPGAU UFF - Open data portal for Urban Resarch,https://portaldados.ppgau.uff.br/,Plano C and PPGAU-UFF,Universidade Federal Fluminense,2022-01-03,Government,Open data portal customized for urbanism and urban studies researchers.,"brazil,academic,urbanism",cc,,License information on the portal is on a per dataset basis. Most datasets have no licensing information.,"RJ, Brazil","-22.90501, -43.13053",BR,pt,active,2022-06-19T23:58Z,CKAN,https://portaldados.ppgau.uff.br/api/3,CKAN API,
+open-data-nepal,"Open Data Nepal",https://opendatanepal.com/,Open Knowledge Nepal,Open Knowledge Nepal,,,"A civil society initiative to make Nepal’s data accessible online perpetually. The portal allows users to publish, explore, download and use data to make their own visualisations, apps and services without any restriction.",ckan nepal,cc-by,https://creativecommons.org/licenses/by/4.0/,,"Kathmandu, Nepal","27.7097,85.3209",NP,np,active,,ckan,https://opendatanepal.com/api,CKAN API,
```

Comment:
```suggestion
open-data-nepal,"Open Data Nepal",https://opendatanepal.com/,Open Knowledge Nepal,Open Knowledge Nepal,,Community,"A civil society initiative to make Nepal’s data accessible online perpetually. The portal allows users to publish, explore, download and use data to make their own visualisations, apps and services without any restriction.",ckan nepal,cc-by,https://creativecommons.org/licenses/by/4.0/,,"Kathmandu, Nepal","27.7097,85.3209",NP,np,active,,ckan,https://opendatanepal.com/api,CKAN API,
```

If this is a community effort, we should probably use the publisher classification "Comminity" (see issue #271).

Reply:
Oh! Got it. Didn't know that.